packages/typespec-ts/src/transform/transformPaths.ts

```diff
@@ -32,12 +37,15 @@ import {
   sortedOperationResponses
 } from "../utils/operationUtil.js";
 import { SdkContext } from "../utils/interfaces.js";
+import { SchemaContext } from "@azure-tools/rlc-common";
 
 export function transformPaths(
-  program: Program,
+  importDetails: Imports,
   client: SdkClient,
   dpgContext: SdkContext
 ): Paths {
```

Comment:
can we remove the program parameters in all the operations that take both program and sdkcontext as parameters at once, seeing this change for many times ? can you also change the parameter signature order as dpgContext, client, importDetails so that it's aligned with other places ?

Reply:
let me do this in a spereated pr